Files/powerGate/Modules/Communication.psm1

```diff
@@ -2,12 +2,43 @@
 Import-Module powerGate
 
 #TODO: configure the powerGate server url and port
-$powerGateServerName = $ENV:Computername
+$powerGateServerName = getRelatedPGServerName
 $powerGateServerPort = "8080"
+# define here witch Vault is used for test and witch one for production
+$testVaults = @(
+	"TestVault1",
+	"TestVault2"
+	# ,...
+)
+$productiveVaults = @(
+	"ProdVault1",
+	"ProdVault2"
+	# ,...
+)
+
+# define here witch Powergate Server is used for test and witch one for production
+$PGServerDefinitions = @{
+	"TEST" = "TestERP";
+	"PROD" = "ProductiveERP"
+}
 $powerGateServerErpPluginUrl = "http://$($powerGateServerName):$($powerGateServerPort)/coolOrange/ErpServices"
 #$powerGateServerErpPluginUrl = "http://$($powerGateServerName):$($powerGateServerPort)/coolOrange/DynamicsNav"
 # Dynamics NAV 2017 Plugin available here: https://github.com/coolOrangeLabs/powergate-dynamics-nav-sample/releases
 
+function getRelatedPGServerName {
+	$connectedVault = $vaultConnection.Vault
+	if ($connectedVault -in $testVaults){
+		return $PGServerDefinitions["TEST"]
+	}
+	elseif ($connectedVault -in $productiveVaults){
+		return $PGServerDefinitions["PROD"]
+	}
+	else {
+		ShowMessageBox -Message "The current connected VAULT $($vaultConnection.Vault) is not mapped in the configuration for any ERP.`nChange the configuration and restart vault!" -Icon Error | Out-Null
```

Comment:
ITs very good to have this erorr here, when the current vault is not configured.
But in **this case** it should not have a default value, because in a real project I always need do remove it.